frontend/src/components/dashboard/ActivityRow.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <tr class="row">
+    <th style="text-align: left" class="tabular-nums" scope="row">
+      <span class="baseline-center"
+        ><span class="smaller">{{ scheduleEntry.number }}</span></span
+      >
```

Comment:
Yes, I didn't really see an advantage in that component. I felt like the code was simpler without it. It was only used once, and if it was intended for reuse, it shouldn't live in components/dashboard/.

Maybe I didn't understand all the benefits of that component? As far as I could tell it only added an inline flex and optionally a smaller font size. Seemed trivial enough for utility classes to me.

Reply:
Actually I'm not even sure the inline flex has any noticeable effect. Maybe I could simplify this altogether?

Reply:
It vertically aligns content to an baseline 1em inline element. This is achieved with a hidden (but not ignored) character in the `::before` pseudo. Because it is the first element inside the inline flex, it is used to align to the inline context. The align center then makes sure that all following content is centered to that hidden character.

I'd like to keep this element, if possible.

Reply:
I see, I tried it out and there is a difference (previously I didn't notice the misalignment even though I tried to see it):
![Screenshot 2022-11-09 at 21-19-02 eCamp3](https://user-images.githubusercontent.com/7566995/200936212-45716f7e-75c7-4d5f-a389-c6a63ddb0c9a.png)
(left activity number is without the component, right activity number is with it)

I added the component back, but renamed it and refactored it to a form where I would actually use it. The `smaller` prop and nested `.content` span are gone because those concern the content rather than the alignment, and they are things I want to control as a user of that component. I also renamed it to `<TextAlignBaseline>`, to more clearly communicate what the component is used for.